### PR TITLE
updated the documentation

### LINKED ISSUE #1783: Improve incremental docs to replace deprecated dlt.current.state() function

### PR DESCRIPTION
**Pull request by dat-a-man**
### Description
Updated the incremental docs

### Related Issues

- Fixes #...
- Closes #1783
- Resolves #...

### CODE REVIEW

File: docs/website/docs/general-usage/incremental-loading.md [hunk 1]
```diff
@@ -1064,7 +1064,7 @@ def tweets():
     data = get_data(start_from=last_val)
     yield data
     # change the state to the new value
-    dlt.current.state()["last_updated"]  = data["last_timestamp"]
+    dlt.current.resource_state()["last_updated"]  = data["last_timestamp"]
```
Comment (AstrakhantsevaAA) [inline docs/website/docs/general-usage/incremental-loading.md:1067]:
```suggestion
    dlt.current.resource_state()["last_updated"] = data["last_timestamp"]
```

**Review by AstrakhantsevaAA — COMMENTED** [review]
can you also update the same thing here: https://dlthub.com/docs/general-usage/incremental-loading#advanced-state-usage-tracking-the-last-value-for-all-search-terms-in-twitter-api

File: docs/website/docs/general-usage/incremental-loading.md (hunk 1 above)
Comment (dat-a-man) [reply docs/website/docs/general-usage/incremental-loading.md:1067]: deleted the extra space

Comment (dat-a-man) [reply docs/website/docs/general-usage/incremental-loading.md]: deleted the extra space

**Review by AstrakhantsevaAA — APPROVED** [review]
good!!

**Pull request merged**